CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # [Unreleased](https://github.com/pybamm-team/PyBaMM/)
 
 ## Features
+- Added a 3D temperature in existing Basic SPM model with two way coupling. ([#5112](https://github.com/pybamm-team/PyBaMM/pull/5112))
```

Comment:
Can you update this to clarify that it added a new `BasicSPM_with_3DThermal` model